modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/RasDetailsQueryParams.java

```diff
@@ -24,4 +26,17 @@ public boolean isMethodDetailsExcluded(String detailParam) {
 
     }
 
+    public boolean isParamSupported(String queryParam) {
```

Comment:
The SupportedQueryParameterNames class has an `isSupported` method... wonder if we can re-use it here?